posts/2023-08-22-ArgoCD-Configuration-Drift.adoc

```diff
@@ -0,0 +1,156 @@
+Click Create in the top left.
+[.img_border_light]
+image::/img/blog/configuration-drift-4.png[Setting the Repo,width=70%,align="center"]
+
+== Syncing DayTrader ==
+On the main page of the Argo CD console, you should see a new tile that looks like this:
+[.img_border_light]
+image::/img/blog/configuration-drift-5.png[Argo Dashboard,width=70%,align="center"]
+Click the Sync button, then on the window that appears and click Sync again.
+Click on the daytrader7 tile to view the app dashboard and sync progress.
+Over the course of a few minutes, you should see resources being created and the app dashboard looking like this:
+[.img_border_light]
+image::/img/blog/configuration-drift-6.png[App deploying,width=70%,align="center"]
+Now the app is deployed and can be reached in a route created under the daytrader7 namespace. Find the URL via oc get routes –n daytrader7, then paste the URL in a browser.
+And there we go! The app is fully deployed and ready to be used.
+[.img_border_light]
+image::/img/blog/configuration-drift-7.png[Daytrader Home,width=70%,align="center"]
+== Demo Configuration Drift ==
+Now that we have the app deployed let's show a scenario where configuration drift occurs and how we can easily correct it.  Let's say a developer changes the memory settings in the configuration file for a quick test.  With the Kubernetes CLI or console change the memory requests from `1024Mi` to `2048Mi` if this resource: `deploy/daytrader7-deploy.yaml`.  In a few moments the argo console should show it is out of sync:
+[.img_border_light]
+image::/img/blog/configuration-drift-outofsync.png[Daytrader Home,width=70%,align="center"]
+Since we have Auto Sync disabled we have to manually tell Argo to update our deployments.  Click on `Sync` button to return the configuration to match what it git says it should be.
+
+[.img_border_light]
+image::/img/blog/configuration-drift-synced.png[Daytrader Home,width=70%,align="center"]
+
+// // // // // // // //
+// LINKS
+//
+// OpenLiberty.io site links:
+// link:/guides/microprofile-rest-client.html[Consuming RESTful Java microservices]
+// 
+// Off-site links:
+// link:https://openapi-generator.tech/docs/installation#jar[Download Instructions]
+//
+// // // // // // // //
+
+== Next Steps ==
+In this blog, we've walked through the practical application of GitOps, showcasing its potential in preventing configuration drift. Its benefits, particularly in maintaining deployment consistency, are evident. If you're considering enhancing your deployment approach or starting a new project, we recommend trying out GitOps for your application. You can extend this approach to deploy operators and other dependencies for you applications.
+
+And for those looking to create or update applications, the open liberty guides available at openliberty.io offer helpful insights and steps.
```

Comment:
May be nice to extend this sentence so devs know what these guides can offer. Also worth adding in a link directly to the guides page and to the OL website overall where mentioned.